server/controllers/duels.js

```diff
@@ -391,11 +391,20 @@ function createShuffledQuestionTypesArray() {
  */
 async function getDuel(id, username) {
   const res = await queryPromise("CALL getDuel(?,?);", [id, username]);
-
   if (res[0].length === 0) {
     return null;
   }
-  return formatDuel(res[0], username);
+
+  const TTL = await (async function () {
+    const res = await queryPromise(
+      "SELECT `value` FROM `server_informations` WHERE `key` = 'config_duel_rounds_per_duel';"
+    );
+    return res[0].value;
+  })();
```

Comment:
You could use something like this
```suggestion
  const TTL = (await queryPromise( "SELECT `value` FROM `server_informations` WHERE `key` = 'config_duel_rounds_per_duel';"
    ))[0].value;


```

Reply:
Trop smart as we say in France